config/config.yaml

```diff
@@ -0,0 +1,95 @@
+languages:
```

Comment:
Changing from `config.json` to `config.yaml` is just a format change, or there's something different/new in the configuration?

Reply:
Updating to version 1.0.0 of `apigentools` determined a "migration" of the json configuration file to a yaml with a new format, [Here the related documentation about the new config format](https://apigentools.readthedocs.io/en/latest/spec_repo/#configconfigyaml).

Additionally this new version does not require an installation of the wrapped Open API tooling as it uses an ad-hoc container that packages the tooling.